engine/requirements.txt

```diff
@@ -34,7 +34,7 @@ cachetools==4.2.2
     # via
     #   google-auth
     #   python-telegram-bot
-celery[redis]==5.3.1
```

Comment:
I don't quite understand why it's dropping the `[redis]` portion here 🤔

Reply:
FYI, https://github.com/astral-sh/uv/issues/1595 (I don't have a strong opinion to preserve the extra, though)

Reply:
thanks for the reference 👍 (seems like including it/excluding it doesn't make a big difference)